homeassistant/components/co2signal/coordinator.py

```diff
@@ -0,0 +1,90 @@
+"""DataUpdateCoordinator for the co2signal integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from datetime import timedelta
+import logging
+from typing import Any, cast
+
+import CO2Signal
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_LATITUDE, CONF_LONGITUDE, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import CONF_COUNTRY_CODE, DOMAIN
+from .exceptions import APIRatelimitExceeded, CO2Error, InvalidAuth, UnknownError
+from .models import CO2SignalResponse
+
+PLATFORMS = [Platform.SENSOR]
```

Comment:
`PLATFORMS` isn't used in this module.

Reply:
Thanks a lot, please see https://github.com/home-assistant/core/pull/100592